app/package.json

```diff
@@ -33,6 +33,7 @@
     "shade": "^0.1.1",
     "three": "0.139",
     "three-stdlib": "^2.17.3",
+    "use-deep-compare-effect": "^1.8.1",
```

Comment:
Object equality is becoming harder to check. Adding this to make future dev easier.